hasura.planx.uk/tests/flows.test.js

```diff
@@ -140,6 +140,57 @@ describe("flows and operations", () => {
       expect(i.mutations).not.toContain("update_published_flows");
     });
   });
+  describe("demoUser", () => {
```

Comment:
```suggestion
  });
  
  describe("demoUser", () => {
```

nit: Always nice to have a newline between these `describe()` blocks for readability.